pkg/util/slo_utils.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/koordinator-sh/koordinator/apis/extension"
+)
+
+// NOTE: functions in this file can be overwritten for extension
+
+const (
+	DefaultPriorityClass = extension.PriorityProd
+	DefaultQoSClass      = extension.QoSLS
+)
+
+var (
+	// DefaultPriorityClassWithQoS is the default PriorityClass according to its QoSClass when the pod does not specify
+	// a PriorityClass explicitly.
+	// NOTE: The pod with PriorityClassNone is considered as PriorityClassProd, and the pod with QoSNone is considered
+	// as qos=LS. A special case is that the pod with PriorityClassNone+QoSBE is considered as PriorityBatch to get
+	// compatible to with existing BE pods.
+	DefaultPriorityClassWithQoS = map[extension.QoSClass]extension.PriorityClass{
```

Comment:
Why not move the file to package "apis/extensions"? Should we replace all code that cares about PriorityClass and QoS ?

Reply:
OK